pom.xml

```diff
@@ -19,6 +19,11 @@
     </properties>
 
     <dependencies>
+        <!-- Azure -->
+        <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-stream-binder-servicebus</artifactId>
```

Comment:
Is this dependency introducing netty library? if yes, consider to set the necessary changes to handle CVE